go.mod

```diff
@@ -111,3 +111,5 @@ replace (
 
 	kubevirt.io/client-go => ./staging/src/kubevirt.io/client-go
 )
+
+go 1.13
```

Comment:
I think we still are using 1.12?

Reply:
I agree with Daniel. 
It is a good practice to not commit the go version. 
I know it can be tedious since IDE is adding it automatically. 
But it fixes the version and causes potential complications.